src/operator/subgraph/tensorrt/nnvm_to_onnx.cc

```diff
@@ -157,6 +157,12 @@ std::string ConvertNnvmGraphToOnnx(
   return serialized_onnx_graph;
 }
 
+void ConvertIdentity(NodeProto* node_proto, const NodeAttrs& attrs,
```

Comment:
Any idea if TRT actually optimizes this out?  I've seen this in a few prod services :-/

Reply:
I believe this should be optimized by ONNX-TRT